specification/apimanagement/resource-manager/Microsoft.ApiManagement/stable/2022-08-01/apimapis.json

```diff
@@ -5363,6 +5363,19 @@
           {
             "$ref": "./apimanagement.json#/parameters/ApiIdParameter"
           },
+          {
+            "name": "$filter",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "|     Field     |     Usage     |     Supported operators     |     Supported functions     |</br>|</br>"
```

Comment:
The description provided is not particularly helpful. Please use this field to describe what this parameter is/does.